kcbq-connector/src/main/java/com/wepay/kafka/connect/bigquery/config/BigQuerySinkConfig.java

```diff
@@ -458,6 +458,13 @@ public class BigQuerySinkConfig extends AbstractConfig {
           + "older than the specified partition expiration time will be permanently deleted. "
           + "Existing tables will not be altered to use this partition expiration time.";
 
+  public static final String GCP_CONNECTOR_USER_AGENT_CONFIG = "bigQueryUserAgent";
```

Comment:
This does not really seem like a UserAgent but more like a "Kafka Connect Runtime Provider" which is then used to construct a UserAgent.  wdyt?

Reply:
@kpatelatwork You are correct. The more appropriate naming should be provider. I will change this.

Reply:
Changed naming.